### PR TITLE
Add more OCP versions for preflight jobs

### PR DESCRIPTION
**Pull request by mrhillsman**
Signed-off-by: Melvin Hillsman <mhillsma@redhat.com>

### CODE REVIEW

**Review by amisstea — APPROVED** [review]

File: core-services/prow/02_config/_plugins.yaml [hunk 1]
```diff
@@ -804,12 +804,36 @@ config_updater:
...
         - ci
       gzip: true
       name: ci-operator-misc-configs
+    ci-operator/config/redhat-openshift-ecosystem/preflight/*-preflight-ocp-4.9.yaml:
+      clusters:
+        app.ci:
+        - ci
+      gzip: true
+      name: ci-operator-misc-configs
+    ci-operator/config/redhat-openshift-ecosystem/preflight/*-preflight-ocp-4.10.yaml:
```
Comment (petr-muller) [inline core-services/prow/02_config/_plugins.yaml:831]: Is there a reason why all these patterns aren't a single `ci-operator/config/redhat-openshift-ecosystem/preflight/*-preflight-ocp-*.yaml` key? That way you would not need our approval when adding new configs.

Same with job CMs below

Comment (mrhillsman) [reply core-services/prow/02_config/_plugins.yaml:831]: I believe we tried that before and got failures. I will try again.

**Review by jomkz — APPROVED** [review]
/lgtm

**Review by hongkailiu — APPROVED** [review]

**Pull request merged**